canu/backup/network/network.py

```diff
@@ -38,6 +38,89 @@
 from canu.style import Style
 from canu.utils.inventory import inventory
 
+
+def backup_switches(online_hosts, folder, unsanitized=False):
+    """Backup switch configurations to specified folder.
+
+    Args:
+        online_hosts: Nornir inventory with online switches
+        folder: Directory to save backup files
+        unsanitized: Whether to retain sensitive data
+
+    Returns:
+        List of saved filenames
+    """
+    # Sanitization filters to remove sensitive data
+    sanitize_filters = [
+        {
+            "regex": r"()(?<=ciphertext).+",
+            "replace": r"\1 <removed>",
+        },
+        {
+            "regex": r"()(?<=password 7).+",
+            "replace": r"\1 <removed>",
+        },
+        {
+            "regex": r"()(?<=password).+",
+            "replace": r"\1 <removed>",
+        },
+        {
+            "regex": r"()(?<=md5).+",
+            "replace": r"\1 <removed>",
+        },
+    ]
+
+    def save_config_to_file(hostname, config):
+        if not unsanitized:
```

Comment:
nit: can we change unsanitized to sanitized to make the code easier to read?

Reply:
Fixed.